$OpenBSD: patch-src_playlist_c,v 1.3 2013/01/11 13:20:50 dcoppa Exp $

Fix some layout issues when playing audio only files
(upstream svn revision r2390)

--- src/playlist.c.orig	Fri Jan 11 11:28:40 2013
+++ src/playlist.c	Fri Jan 11 11:29:10 2013
@@ -825,7 +825,7 @@ void menuitem_view_playlist_callback(GtkMenuItem * men
 
     if (remember_loc)
         use_remember_loc = TRUE;
-    adjust_layout();
+    g_idle_add(set_adjust_layout, idledata);
 }
 
 void undo_playlist_sort(GtkWidget * widget, void *data)
